src/Finder/SourceFilesFinder.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Copyright © 2017 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+declare(strict_types=1);
+
+namespace Infection\Finder;
+
+use Symfony\Component\Finder\Finder;
+
+/**
+ * @author Lander Vanderstraeten <lander_vanderstraeten@hotmail.com>
+ */
+class SourceFilesFinder
+{
+    private $sourceDirectories;
+    private $excludeDirectories;
+
+    public function __construct(array $sourceDirectories, array $excludeDirectories)
```

Comment:
I would add a phpdoc bloc to add the `string[]` type, otherwise it might not clear what we are dealing with